emp-wrapper/src/main/java/org/ballerinalang/sf/ListenerUtil.java

```diff
@@ -36,15 +40,13 @@
 import io.ballerina.runtime.internal.values.MapValue;
 import io.ballerina.runtime.internal.values.ObjectValue;
 
-import static org.ballerinalang.sf.LoginHelper.login;
+import static org.ballerinalang.sf.Constants.*;
 
 public class ListenerUtil {
```

Comment:
What this class is supposed to represent?

Reply:
The connector wrapper has a separate class for constants. So we are importing that class and accessing the variables.